### ISSUE TITLE
User Permissions: Add option to explicitly allow all elements (current and future)

### ISSUE THREAD

**Issue by bezin** [contao/contao#6018]
When you configure back-end user permissions, you can allow all currently available options, e.g. allow access to every current image size. If you add a new image size it must be allowed manually.

We would suggest an explicit option "Allow all", which means all current or future image sizes, i.e. no restrictions what so ever. 

Alternatively, everything could be allowed by default and one has to toggle a checkbox first, if you want to limit the accessible elements. However this would be quite some logical change:

![limit_access_to_content_element_types](https://user-images.githubusercontent.com/4400435/236675918-6d72c882-d806-4a72-9b16-c5c6453c4e4e.jpg)

This also applies to everything else in the permissions module: Back-end modules, page types, forms, content elements, form fields, image sizes, calendars, news archives. Whenever you add something new, you have to manually allow it afterwards. 

Discussed in the back-end session at the [Contao Camp 2023](https://github.com/contao-community-alliance/contao-camps/blob/main/2023/camp-2023.md#samstag-06052023).

**Comment by k-webdesign** [comment]
This occurs not only when installing extensions, also when a new news archive, Image size, newsletter,… gets added and this one is missing in the permissions

**Comment by bezin** [comment]
Yes, that was just one example, but it was indeed misleading. I updated the description accordingly.

**Comment by zoglo** [comment]
I think this is implemented with #9012 as you can now add Create, Update and Delete Actions for tables (In this case the image size tables)
